platforms/openal/SoundSystemAL.cpp

```diff
@@ -226,73 +239,76 @@ void SoundSystemAL::update(float x, float y, float z, float yaw)
 	}
 }
 
-void SoundSystemAL::play(const SoundDesc& sound, float x, float y, float z, float volume, float pitch, bool is_ui)
+void SoundSystemAL::playAt(const SoundDesc& sound, float x, float y, float z, float volume, float pitch)
 {
 	// Check
 	if (!loaded)
 	{
 		return;
 	}
 
+	if (volume <= 0.0f)
+		return;
+
 	// Load Sound
 	ALuint buffer = get_buffer(sound);
-	if (volume > 0.0f && buffer)
+	if (!buffer)
+		return;
+	
+	// Get Source
+	ALuint al_source;
+	if (idle_sources.size() > 0)
 	{
-		// Get Source
-		ALuint al_source;
-		if (idle_sources.size() > 0)
-		{
-			// Use Idle Source
-			al_source = idle_sources.back();
-			idle_sources.pop_back();
-		}
-		else
+		// Use Idle Source
+		al_source = idle_sources.back();
+		idle_sources.pop_back();
+	}
+	else
+	{
+		// Create Source
+		alGenSources(1, &al_source);
+		// Special Out-Of-Memory Handling
 		{
-			// Create Source
-			alGenSources(1, &al_source);
-			// Special Out-Of-Memory Handling
+			ALenum err = alGetError();
+			if (err == AL_OUT_OF_MEMORY)
 			{
-				ALenum err = alGetError();
-				if (err == AL_OUT_OF_MEMORY)
-				{
-					return;
-				}
-				else
-				{
-					AL_ERROR_CHECK_MANUAL(err);
-				}
+				return;
+			}
+			else
+			{
+				AL_ERROR_CHECK_MANUAL(err);
 			}
 		}
+	}
 
-		// Set Properties
-		alSourcef(al_source, AL_PITCH, pitch);
-		AL_ERROR_CHECK();
-		alSourcef(al_source, AL_GAIN, volume);
-		AL_ERROR_CHECK();
-		alSource3f(al_source, AL_POSITION, x, y, z);
-		AL_ERROR_CHECK();
-		alSource3f(al_source, AL_VELOCITY, 0, 0, 0);
-		AL_ERROR_CHECK();
-		alSourcei(al_source, AL_LOOPING, AL_FALSE);
-		AL_ERROR_CHECK();
-		alSourcei(al_source, AL_SOURCE_RELATIVE, is_ui ? AL_TRUE : AL_FALSE);
-		AL_ERROR_CHECK();
+	// Set Properties
+	alSourcef(al_source, AL_PITCH, pitch);
+	AL_ERROR_CHECK();
+	alSourcef(al_source, AL_GAIN, volume);
+	AL_ERROR_CHECK();
+	alSource3f(al_source, AL_POSITION, x, y, z);
+	AL_ERROR_CHECK();
+	alSource3f(al_source, AL_VELOCITY, 0, 0, 0);
+	AL_ERROR_CHECK();
+	alSourcei(al_source, AL_LOOPING, AL_FALSE);
+	AL_ERROR_CHECK();
+	alSourcei(al_source, AL_SOURCE_RELATIVE, AL_FALSE);
+	AL_ERROR_CHECK();
 
-		// Set Attenuation
-		alSourcef(al_source, AL_MAX_DISTANCE, 16.0f);
-		AL_ERROR_CHECK();
-		alSourcef(al_source, AL_ROLLOFF_FACTOR, 6.0f);
-		AL_ERROR_CHECK();
-		alSourcef(al_source, AL_REFERENCE_DISTANCE, 5.0f);
-		AL_ERROR_CHECK();
+	// Set Attenuation
+	alSourcef(al_source, AL_MAX_DISTANCE, 16.0f);
+	AL_ERROR_CHECK();
+	alSourcef(al_source, AL_ROLLOFF_FACTOR, 3.0f);
```

Comment:
Why did you change `AL_ROLLOFF_FACTOR`?

Reply:
in https://github.com/ReMinecraftPE/mcpe/pull/34/commits/bc983340d61c9f9351289b0e12e19b4c10357e38
"Adjusted rolloff factor, not sure if this is just placebo but it felt a bit better?"